packages/react-scripts/template/README.md

```diff
@@ -442,27 +441,17 @@ Then we can change `start` and `build` scripts to include the CSS preprocessor c
    }
 ```
 
-Now running `npm start` and `npm run build` also builds Sass files. Note that `node-sass` seems to have an [issue recognizing newly created files on some systems](https://github.com/sass/node-sass/issues/1891) so you might need to restart the watcher when you create a file until it’s resolved.
+Now running `npm start` and `npm run build` also builds Sass files. Note that `node-sass-chokidar` seems to have an issue recognizing newly created files on some systems so you might need to restart the watcher when you create a file until it’s resolved.
```

Comment:
ok, thanks @kellyrmilligan 
can you open an issue on `node-sass-chokidar` and describe steps to reproduce, as well as your system details. (e.g. OS, node, npm, and package versions) thanks so much and I will try to have this fixed ASAP

It works as expected for me so I need to try and duplicate your setup.